### ISSUE TITLE
Opacity<1 wrong color mixing

### ISSUE THREAD

**Issue by 14958159120765182** [armory3d/armorpaint#1733]
ArmorPaint version:1.0
Windows

When you paint with lower than 1 opacity, instead of mixing the color to the bottom color proportional to the brush opacity like all the other similar programs, in here it seems to mix the opacity itself. 

![Screenshot 2024-07-10 094218](https://github.com/armory3d/armortools/assets/112494032/20d418e1-7d65-40e3-89d1-2a3ecb50ee58)

In this example both the layers use the same color, I just used the low opacity brush on the top one. I assume the reason darker color is the fact that the default color is black if there is no layer, so because the mix option mixes the opacity itself it appears darker. 

Basically there is no option for this https://docs.krita.org/en/reference_manual/blending_modes/mix.html#normal blend mode. 

Also when you make a new paint layer, if you use opacity<1 brushes on them, the color doesn't stack if you pass through the same place again.

It would be appreciated if you guys added the linked above blend mode. As a new blend mode perhaps?

**Comment by luboslenco** [comment]
The mixing should be finally resolved now, thanks for report.
